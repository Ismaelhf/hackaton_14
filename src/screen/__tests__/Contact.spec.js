// import { shallow, mount, render } from 'enzyme';
import React from 'react';
import {Text, View, TouchableOpacity, Image} from 'react-native';
import {shallow} from 'enzyme';
import Contact from '../Contact';

const wrapper = shallow(<Contact />);

describe('Contact screeen', () => {
  it('should render', () => {
    expect(wrapper.exists()).toBe(true);
  });
  it('should Contact element', () => {
    expect(wrapper.find(View).length).toBe(3);
    expect(wrapper.find(Image).length).toBe(1);
    expect(wrapper.find(Text).length).toBe(3);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
