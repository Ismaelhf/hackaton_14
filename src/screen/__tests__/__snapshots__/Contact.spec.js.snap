// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact screeen should match snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View>
    <Image
      source={
        Object {
          "uri": "https://pbs.twimg.com/profile_images/794753207306166272/TAEMQXGa_400x400.jpg",
        }
      }
      style={
        Object {
          "height": 300,
          "width": "100%",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginHorizontal": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Numero de contacto: 1125482
    </Text>
    <Text
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Correo: agencia@gmail.com
    </Text>
    <Text
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Direcci√≥n: calle 158 cercado de Lima
    </Text>
  </View>
</View>
`;
